UNIT2Assessment.js

```diff
@@ -0,0 +1,50 @@
+document.addEventListener("DOMContentLoaded", async () => {
+    let select = document.querySelector("select");
+    let form = document.querySelector("form");
+    let userInput = document.querySelector("#userInput");
+    let submit = document.querySelector("submit");
+    let filmInfo = document.querySelector("#filmInfo");
+    let ul = document.querySelector("#submittedReview");
+
+    const getMovie = async () => {
+        try {
+            filmInfo.innerHTML = "";
+            let res = await axios.get("https://ghibliapi.herokuapp.com/films");
+            let movieArr = res.data;
+            movieArr.forEach(movie => {
+                let option = document.createElement("option");
+                option.innerText = movie.title;
+                select.appendChild(option);
+            })
+            return res.data;
+        } catch (error) {
+            console.log(error)
+        }
+    }
+    
+    getMovie();
+
+    select.addEventListener("change", async (e) => {
+        // select.value = e.target.value;
+        let movie = await getMovie();
```

Comment:
There's no need to call the getMovie function again here. It creates an interesting bug - every time the user clicks on any movie title in the select bar , the bar gets the list of the movies again in addition to the movies already there. So the list doubles, triples and so on with each click.

Reply:
You do need to make an axios call again to the same url, just not the full getMovie function